config/config.go

```diff
@@ -1071,9 +1071,16 @@ func (p *PackageConfig) LocalDownloadPath(packagesDir string) string {
 }
 
 // LocalDataParentDirectory returns the folder that will contain the all packages of this type.
```

Comment:
[opt] There is a comment on like 1063 above LocalDataDirectory that still has the .